package dev.vality.disputes.provider.payments.exception;

public class ProviderCallbackStatusWasUpdatedByAnotherThreadException extends RuntimeException {
}
